src/components/search-input/SearchInput.tsx

```diff
@@ -7,7 +7,12 @@ import SearchIcon from '@mui/icons-material/Search'
 
 import { styles } from './SearchInput.styles'
 
-const SearchInput = ({ search, setSearch }) => {
+interface SearchInputProps {
+  search: string
+  setSearch: (state: string) => void
```

Comment:
A better way to set type for `setState`:
`setSearch: React.Dispatch<React.SetStateAction<string>>`